### PR TITLE
Ws  allows admin to approve sightings

### PR DESCRIPTION
**Pull request by dorothyynwong**
- Create a new controller AdminController with route “admin”
- Create a new endpoint in AdminController to approve sightings with [HttpPut(“approve/sighting={sightingId}")]
- Use [Authorize(”Roles=Admin”)] to make sure the endpoint can be accessed by admins only
- Add a new method to SightingsService for changing IsApproved from false to true and update the database

### CODE REVIEW

File: backend/Controllers/AdminController.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Services;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
```
Comment (Kiran-R-K) [inline backend/Controllers/AdminController.cs:7]: Now that the Authorization PR (to check that users are logged in) has been merged in, I think an Authorize attribute should be added here to make sure the user has logged in, even before we check they are an admin

Comment (Kiran-R-K) [reply backend/Controllers/AdminController.cs:7]: I wonder if you can check they are logged in and an admin right up here at the top of the controller? then we won't have to do it separately for each endpoint

**Review by Kiran-R-K — CHANGES_REQUESTED** [review]
REally good work, nice and clean, I've left some comments

File: backend/Controllers/AdminController.cs (hunk 1 above)
Comment (dorothyynwong) [reply backend/Controllers/AdminController.cs:7]: I think one [Authorize(Roles="Admin")] here is enough as it implies a user needs authentication already with it

**Review by Kiran-R-K — APPROVED** [review]
well done

**Pull request merged**